roles/DCOS.agent_windows/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+- name: "Re-run setup to use custom facts"
```

Comment:
Why quoting here and within all other steps not?

Reply:
Removed the handler